remote-config.jsonc

```
@@ -123,6 +123,30 @@
         },
         {
           "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
+        },
+        {
+          "message": "You can contribute your own tips to these daily tips with a PR on https://github.com/ddev/remote-config"
+        },
+        {
+          "message": "DDEV's goal is to make developers happy. We hope you're happy today!"
+        },
+        {
+          "message": "If you're on macOS you may want to try the excellent new Docker provider OrbStack, https://orbstack.dev"
+        },
+        {
+          "message": "Most teams check in the project .ddev directory so everybody has the same configuration. But do NOT force-add the .ddev/.gitignore, which is managed by DDEV."
```

Use backticks consistently, rephrase assuming a team has more than one project:

```suggestion
          "message": "Most teams check in a project’s `.ddev` directory so everybody has the same configuration. But do NOT force-add the `.ddev/.gitignore`, which is managed by DDEV."
```